node.json

```diff
@@ -196,5 +203,25 @@
         "title": "Line Height",
         "ui_width": 4,
         "type": "integer"
+    }, {
+        "default": "de",
+        "name": "language",
+        "title": "Display Language",
+        "type": "select",
+        "options": [
+            ["de", "Deutch"],
+            ["en", "English"],
+            ["cy", "Cymraeg"]
+        ]
+    }, {
+        "name": "timezone",
+        "title": "Timezone",
+        "type": "string",
+        "default": "Europe/Berlin"
+    }, {
+        "name": "nre_powered",
+        "title": "Show National Rail Equiries Logo",
```

Comment:
I think this should be a more generic "show data provider logo", and the logo source should get determined by the provider setting above.

Reply:
Done.